modules/manage/pages/cluster-maintenance/compaction-settings.adoc

```diff
@@ -6,25 +6,77 @@ Configure compaction for your cluster to optimize storage utilization.
+
+You can specify how long Redpanda keeps these tombstones for compacted topics using both a cluster configuration property config_ref:tombstone_retention_ms,true,properties/cluster-properties[] and a topic configuration property xref:reference:properties/topic-properties.adoc#deleteretentionms[`delete.retention.ms`]. If both are set, the topic-level tombstone retention policy overrides the cluster-level policy.
+
+[NOTE]
+====
+Redpanda does not currently remove tombstone records for compacted topics with Tiered Storage enabled. 
+
+You cannot enable `tombstone_retention_ms` if you have enabled any of the Tiered Storage cluster properties `cloud_storage_enabled`, `cloud_storage_enable_remote_read`, and `cloud_storage_enable_remote_write`.
+
+On the topic level, you cannot enable `delete.retention.ms` at the same time as the Tiered Storage topic configuration properties `redpanda.remote.read` and `redpanda.remote.write`.
+====
+
+To set the cluster-level tombstone retention policy, run the command:
+
+[,bash]
+----
+rpk cluster config set tombstone_retention_ms=100
+----
+
+You can unset the tombstone retention policy for a topic so it inherits the cluster-wide default policy:
+
+[,bash]
+----
+rpk topic alter-config <topic-name> --delete delete.retention.ms
+----
+
+To override the cluster-wide default for a specific topic:
+
+[,bash]
+----
+rpk topic alter-config <topic-name> --set delete.retention.ms=5
+----
+
+To disable tombstone removal for a specific topic:
+
+[,bash]
+----
+rpk topic alter-config <topic-name> --set delete.retention.ms=-1
+----
+
+Redpanda removes tombstones as follows:
+
+* For topics with a `compact` only cleanup policy: Tombstones are removed when the topic exceeds the tombstone retention limit. The `delete.retention.ms` or `tombstone_retention_ms` values therefore also set the time bound that a consumer has in order to see a complete view of the log with tombstones present before they are removed.
+* For topics with a `compact,delete` cleanup policy: Both the tombstone retention policy and standard garbage collection can remove tombstone records.
+
+If obtaining a complete snapshot of the log, including tombstone records, is important to your consumers, set the tombstone retention value such that consumers have enough time for their reads to complete before tombstones are removed. Consumers may not see tombstones if their reads take longer than `delete.retention.ms` and `tombstone_retention_ms`. The trade-offs to ensuring tombstone visibility to consumers are increased disk usage and potentially slower compaction. 
+
+On the other hand, if more frequent cleanup of tombstones is important for optimizing workloads and space management, consider setting a shorter tombstone retention, for example the typical default of 24 hours (86400000 ms). Consequently, if you set tombstone retention limits that are too low, it could cause more data inconsistencies.
```

Comment:
> Consequently, if you set tombstone retention limits that are too low, it could cause more data inconsistencies.

"Data inconsistencies" sounds a bit too scary- consumers not seeing tombstones isn't exactly an "inconsistency".

I think you illustrated the trade-off well enough in the paragraph above on L77, I would recommend removing this sentence or rewording it.